### ISSUE TITLE
Unable to load asset on iOS

### ISSUE THREAD

**Issue by bottee** [jerson/flutter-rsa#16]
If I try to run my debug flutter application on my device, I get following error on iOS:

`Caught error: Unable to load asset: /private/var/containers/Bundle/Application/E8B1597B-8CCB-4A34-B147-48E683C039C9/Runner.app/assets/locale/locale_de.json`

If I remove the entry 

`fast_rsa: ^1.3.6`

from my `pubspec.yaml` and all related source code  everything works fine. On Android Fast RSA works without this issue.

**Comment by jerson** [comment]
hi @bottee thanks for report the issue, sound a bit weird but can be related, if you are using null safety can you try with latest version please?
https://pub.dev/packages/fast_rsa/versions/2.0.0-nullsafety.0

can you post the content of this locale file please?

**Comment by bottee** [comment]
Hi @jerson,

thank you very much for the fast response 👍 

I tried to understand the problem more and I found the reason. For testing my application I need a changed prefix for loading strings from assets. To determine if I'm testing I used `Directory.current.path`. It seems that your plug-in changes the current directory away from `'/'`.

Now I use `Platform.environment.containsKey('FLUTTER_TEST')`. Now everything works again.

I do not know why your plug-in changes the current directory and if this has any other potential side effects.

In any case, sorry for the inconvenience. And thank you very much for your great plug-in 👍 

From my side, the issue can be closed.
